scripts/run_validator.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
-tag=latest
+tag=introduce-questionnaire-flow
```

Comment:
Update before merge.
```suggestion
tag=v2.0.0
```